doc/source/performance/performance-factors.rst

```diff
@@ -193,6 +193,15 @@ VFS parallelism -- ``vfs.min_parallel_size`` and ``vfs.file.max_parallel_ops``
     the maximum number of parallel operations for ``file:///`` URIs, independently
     of the thread pool size, allowing you to over- or under-subscribe VFS threads.
 
+VFS read batching -- ``vfs.max_batch_read_size`` and ``vfs.max_batch_read_amplification``
```

Comment:
Shall we mention here that the default (`1.0`) amplification factor will only allow batching adjacent ranges?